docs/catalog/sinks/telegram.rst

```diff
@@ -0,0 +1,78 @@
+Telegram
+#################
+
+Robusta can send playbook results to Telegram conversations.
+
+.. note::
+
+    ``TableBlock`` is omitted from Telegram sinks because it's too big for most Telegram chat clients.
+
+    In addition, 2-way interactivity (``CallbackBlock``) isn't implemented yet.
+
+Get your api_id and api_hash
+------------------------------------------------
+In order to run a the Telegram sink, you'll need an ``api_id`` and ``api_hash``
+
+It's easy!
```

Comment:
No need. Just show how easy it is instead of claiming that.

Reply:
removed